nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -676,6 +677,14 @@ in
           Defines a group of servers to use as proxy target.
         '';
         default = {};
+        example = {
```

Comment:
May be better to use `literalExample` here: this has better rendering in the manual.